src/App.vue

```diff
@@ -207,6 +214,7 @@ export default class App extends Mixins(BcolMixin, DateMixin, FilingTemplateMixi
   private haveData: boolean = false
   private saveErrors: Array<object> = []
   private saveWarnings: Array<object> = []
+  private fileAndPayInvalidNameRequestDialog: boolean = false
```

Comment:
You may have to add this flag to isErrorDialog() and resetFlags().

Reply:
Done